README.md

```diff
@@ -29,9 +29,9 @@ One of the ways you can help the project is by collecting data to train our mode
 
 Clone the repo and run:
 
-`npm install`
+1. `npm install`
 
-`npm start`
+2. `npm start` or `npm run dev` (like `npm start` but the server automatically restarts after files are changed)
```

Comment:
re "files are changed" - is this any file? Or any `.js` file?

Reply:
Ooh, good catch, I've specified which ones (i.e. any HTML/CSS/JS files).